sdk/mobile-center/src/main/java/com/microsoft/azure/mobile/MobileCenter.java

```diff
@@ -491,8 +491,10 @@ private synchronized void setInstanceEnabled(boolean enabled) {
             mUncaughtExceptionHandler.unregister();
         }
 
-        /* Update state. */
```

Comment:
Had a conversation offline. `setInstanceEnabled` is looking at the storage to determine the core is disabled or enabled while core state is changing. We do need this.